charts/merlin/templates/kaniko-service-account.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.merlin.imageBuilder.serviceAccount .Values.merlin.imageBuilder.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.merlin.imageBuilder.serviceAccount.name }}-{{ .Release.Name }}
```

Comment:
nit: Is there any reason not to use full name?

Reply:
Hang on, i think this is wrong, will change to just use 
```
{{ .Values.merlin.imageBuilder.serviceAccount.name }}
```
without the `{{ .Release.Name }}`

Reply:
But without the release name, there could be conflict of service account when there is multiple merlin release in same namespace?

Reply:
Yeah that was my initial thought, but then when specifying service account specified externally it should respect the name provided.